testing/browser-functional/nightwatch/requirements.js

```diff
@@ -0,0 +1,38 @@
+const { DEFAULT_BROWSER, getFlag, getBrowser, logs, isBrowserInstalled, isBotRunning } = require('./utils');
```

Comment:
Should we include the MS header?

Reply:
It didn't have the header before, I added it, thanks!